hack/local-up-karmada.sh

```diff
@@ -29,12 +29,18 @@ KIND_LOG_FILE=${KIND_LOG_FILE:-"/tmp/karmada"}
 #step0: prepare
 # proxy setting in China mainland
 
-if [[ -n ${CHINA_MAINLAND:-} ]]; then
-export GOPROXY=https://goproxy.cn # set go proxy
+if [[ ! -n ${CHINA_MAINLAND:-} ]]; then
+export GOPROXY=https://goproxy.cn,direct # set domestic go proxy
 # set mirror registry of k8s.gcr.io
-sed -i'' -e "s#k8s.gcr.io#registry.aliyuncs.com/google_containers#g" artifacts/deploy/karmada-etcd.yaml
-sed -i'' -e "s#k8s.gcr.io#registry.aliyuncs.com/google_containers#g" artifacts/deploy/karmada-apiserver.yaml
-sed -i'' -e "s#k8s.gcr.io#registry.aliyuncs.com/google_containers#g" artifacts/deploy/kube-controller-manager.yaml
+sed -i'' -e "s#k8s.gcr.io#registry.aliyuncs.com/google_containers#g" ${REPO_ROOT}/artifacts/deploy/karmada-etcd.yaml
+sed -i'' -e "s#k8s.gcr.io#registry.aliyuncs.com/google_containers#g" ${REPO_ROOT}/artifacts/deploy/karmada-apiserver.yaml
+sed -i'' -e "s#k8s.gcr.io#registry.aliyuncs.com/google_containers#g" ${REPO_ROOT}/artifacts/deploy/kube-controller-manager.yaml
+# set mirror registry in the dockerfile of components of karmada
```

Comment:
Is there a good way to remind us to update the alpine repository URL if the `from image` changes?
I mead if the `FROM alpine:3.7` changed in Dockerfile we must edit  `http://mirrors.ustc.edu.cn/alpine/v3.7/main/` synchronously.

Reply:
Add a note in Dockerfile？Or if only the version of alpine will change, it can be solved by setting a varible to record the version of alpine.

Reply:
I don't think it should be noted in `Dockerfile`.
Thanks for @lfbear pointing it out, it may happen, but it is not a big deal, we don't update this frequently.